propka/bonds.py

```diff
@@ -32,11 +26,11 @@ def __init__(self):
         # predefined bonding distances
         self.distances = {'S-S' : DISULFIDE_DISTANCE, 'F-F' : FLUORIDE_DISTANCE}
         self.distances_squared = {}
-        for k in self.distances.keys():
-            self.distances_squared[k] = self.distances[k] * self.distances[k]
-        self.H_dist = HYDROGEN_DISTANCE
+        for key in self.distances:
+            self.distances_squared[key] = self.distances[key] * self.distances[key]
+        self.h_dist = HYDROGEN_DISTANCE
```

Comment:
In the other pull requests @orbeckst raised the issue about API breaking changes when renaming publicly visible names. That being said, a simple google search didn't find any third party scripts which use these, so it's probably find to rename these.

In that case, instead of renaming `self.H_dist`, I would remove it because it's not used outside this function, only `h_dist_squared` is used.

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/db8571414e01b7451790c54450f90aa875cbd180

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/02ca72d91d52838ee4ee2fe978b5dd3bc4a75074